src/runtime/pkg/katautils/config.go

```diff
@@ -96,6 +97,7 @@ type hypervisor struct {
 	GuestHookPath                  string   `toml:"guest_hook_path"`
 	GuestMemoryDumpPath            string   `toml:"guest_memory_dump_path"`
 	SeccompSandbox                 string   `toml:"seccompsandbox"`
+	RemoteHypervisorSocket         string   `toml:"remote_hypervisor"`
```

Comment:
Wouldn't it be clearer to call the TOML key simply `remote_hypervisor_socket`?

Reply:
I agree. I'll change it to `remote_hypervisor_socket`

Reply:
I renamed the option name to `remote_hypervisor_socket`, and also rebased the branch to the latest `CCv0` branch.

Reply:
@yoheiueda Thanks!